src/rpdk/core/test.py

```diff
@@ -52,14 +54,61 @@ def temporary_ini_file():
         yield str(path)
 
 
-def get_overrides(root):
+def get_cloudformation_exports(region_name, endpoint_url):
+    session = create_sdk_session(region_name)
+    cfn_client = session.client("cloudformation", endpoint_url=endpoint_url)
+    next_token = ""
+    exports = {}
+    while next_token is not None:
+        result = (
+            cfn_client.list_exports(NextToken=next_token)
+            if next_token != ""
+            else cfn_client.list_exports()
+        )
+        exports = {
+            **exports,
+            **{export["Name"]: export["Value"] for export in result["Exports"]},
+        }
+        next_token = None if "NextToken" not in result else result["NextToken"]
+    return exports
+
+
+def render_jinja(overrides_raw, region_name, endpoint_url):
+    env = Environment(autoescape=True)
+    parsed_content = env.parse(json.dumps(overrides_raw))
+    variables = meta.find_undeclared_variables(parsed_content)
+    if variables:
+        exports = get_cloudformation_exports(region_name, endpoint_url)
+        invalid_exports = []
+        var_map = {}
+        for variable in variables:
+            if variable in exports:
+                var_map[variable] = exports[variable]
+            else:
+                invalid_exports.append(variable)
+        if invalid_exports:
+            invalid_exports_message = (
+                "Override file invalid: %s are not valid cloudformation exports."
+                + "No Overrides will be applied"
+            )
+            LOG.warning(invalid_exports_message, invalid_exports)
+            return empty_override()
+        overrides_template = Template(json.dumps(overrides_raw))
+        to_return = json.loads(overrides_template.render(var_map))
+    else:
+        overrides_template = Template(json.dumps(overrides_raw))
+        to_return = json.loads(overrides_template.render())
```

Comment:
not sure how lightweight the render operation is, but shouldn't this just be overrides_raw? could save the render/load

Reply:
Was thinking about if they wanted to just use jinja in general. But thinking about it again, I don't see a use cause where they wouldn't use variables anyway. I'll remove it

Reply:
At least for that we'd need a way to pass in the variables. Do you think that'd be feasible or worth doing? I don't really have an opinion

Reply:
Until we have a strong use case, I don't think it's worth doing. It would require api changes to pass in custom parameters. If we don't want that, we'd have to let them put the inputs in another text file, which is what the overrides.json is meant for in the first place. I removed it